ta/remoteproc/sub.mk

```diff
@@ -0,0 +1,6 @@
+# REMOTEPROC TA heap size can be customized if 4kB is not enough
+CFG_REMOTEPROC_TA_HEAP_SIZE ?= (4 * 1024)
```

Comment:
No, CFG variables should not change once we start to process the `sub.mk` files.

Reply:
This is inspired from the PKCS11 TA : [`ta/pkcs11/sub.mk`](https://github.com/OP-TEE/optee_os/blob/master/ta/pkcs11/sub.mk#L7)

Please, could you provide some advice on how to accomplish this in a clean manner?

Reply:
Ouch, I didn't see that one before. I'd prefer to put this in `mk/config.mk`.

Reply:
I also prefer not setting configuration variables in the `sub.mk` files and use them only to list the files to be compiled.

On the other hand it is convenient to keep the TA config under the TA source directory (i.e., `ta/remoteproc`). How about doing that in `ta/remoteproc/user_ta.mk`? It is included here: https://github.com/OP-TEE/optee_os/blob/3.22.0/ta/mk/build-user-ta.mk#L25. We could also introduce `ta/remoteproc/config.mk` and include it before `user_ta.mk`.

Reply:
It's still somewhat late in the middle of sub.mk loading for TAs. I don't suppose we need to make a big problem out of this. So if you prefer this fine with me for now.